cmd/krew/cmd/list.go

```diff
@@ -41,41 +40,30 @@ Plugins will be shown as "PLUGIN,VERSION"`,
 				return errors.Wrap(err, "failed to find all installed versions")
 			}
 			if !(isatty.IsTerminal(os.Stdout.Fd()) || isatty.IsCygwinTerminal(os.Stdout.Fd())) {
-				fmt.Fprintf(os.Stdout, "%s\n", strings.Join(sortedKeys(plugins), "\n"))
+				for plugin := range plugins {
+					fmt.Fprintln(os.Stdout, plugin)
+				}
 				return nil
 			}
-			printAlignedColumns(os.Stdout, "PLUGIN", "VERSION", plugins)
-			return nil
+			var table [][]string
+			for p, version := range plugins {
```

Comment:
The plugins should be sorted here (got removed from the tableWriter).

Reply:
> The plugins should be sorted here (got removed from the tableWriter).

I removed on purpose, because the method actually returns a sorted list anyway?